ext/src/ruby_api/store.rs

```diff
@@ -170,6 +172,64 @@ impl Store {
         Ok(())
     }
 
+    /// @yard
+    /// Sets limits on the {Store}.
+    /// @param memory_size [Integer]
+    ///   The maximum number of bytes a linear memory can grow to.
+    /// @param table_elements [Integer]
+    ///   The maximum number of elements in a table.
+    /// @param instances [Integer]
+    ///   The maximum number of instances that can be created for a Store.
+    /// @param tables [Integer]
+    ///   The maximum number of tables that can be created for a Store.
+    /// @param memories [Integer]
+    ///   The maximum number of linear memories that can be created for a Store.
+    /// @raise [Error] if arguments can not be converted.
+    pub fn set_limits(&self, args: &[Value]) -> Result<(), Error> {
+        let args = scan_args::scan_args::<(), (), (), (), _, ()>(args)?;
+        let kw = scan_args::get_kwargs::<
+            _,
+            (),
+            (Option<u64>, Option<u64>, Option<u64>, Option<u64>, Option<u64>),
+            ()
+        >(
+            args.keywords,
+            &[],
+            &["memory_size", "table_elements", "instances", "tables", "memories"]
+        )?;
+
+        let mut limiter: StoreLimitsBuilder = StoreLimitsBuilder::new();
+        let (memory_size, table_elements, instances, tables, memories) = kw.optional;
+
+        if memory_size.is_some() {
```

Comment:
Not sure if there's a more elegant/idiomatic way to deal with all of these `Option` variables.

Reply:
To avoid the the `is_some()` + `unwrap` combination, I'd suggest using the [`if let` pattern](https://doc.rust-lang.org/rust-by-example/flow_control/if_let.html), that's probably a bit more idiomatic.